From dhollis@davehollis.com Mon Dec 19 11:01:27 2005
From: David Hollis <dhollis@davehollis.com>
Subject: USB: asix.c - Add Linksys USB200M Rev 2 ids
To: Greg KH <greg@kroah.com>
Date: Mon, 19 Dec 2005 13:58:38 -0500
Message-Id: <1135018718.26911.3.camel@dhollis-lnx.sunera.com>

Attached patch adds device IDs for the Linksys USB200M Rev 2 device
which uses the AX88772 chipset.

Signed-off-by: David Hollis <dhollis@davehollis.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/asix.c |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/drivers/usb/net/asix.c
+++ gregkh-2.6/drivers/usb/net/asix.c
@@ -912,6 +912,10 @@ static const struct usb_device_id	produc
 	// ASIX AX88772 10/100
         USB_DEVICE (0x0b95, 0x7720),
         .driver_info = (unsigned long) &ax88772_info,
+}, {
+	// Linksys USB200M Rev 2
+	USB_DEVICE (0x13b1, 0x0018),
+	.driver_info = (unsigned long) &ax88772_info,
 },
 	{ },		// END
 };
